### PR TITLE
Carousel Examples: ad note informing authors that aria-rolesdescription uses the language of the page

### LINKED ISSUE #2355: carousel pattern should use localized "aria-roledescription"s for its elements
The [ARIA Authoring Practices Guide *carousel* pattern page](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/) currently gives the following information:

> The carousel container has the [`aria-roledescription`](https://w3c.github.io/aria/#aria-roledescription) property set to **`carousel`**.
> ...
> Each slide container has role group with the property aria-roledescription set to **`slide`**.

But according to [*Accessible Rich Internet Applications (WAI-ARIA) 1.3*](https://www.w3.org/TR/wai-aria-1.2/#aria-roledescription):
> [`aria-roledescription` defines] a human-readable, author-**localized** description for the role of an element.

I understand *localized* description here as *language*-localized. Thus, the given `aria-roledescription`s **carousel** and **slide** are only valid within an English language locale. And indeed, with my computer's German locale, NVDA awkwardly starts spelling "c-a-r-o-u-s-e-l" instead of communicating "Karussell".

Assuming I understood `aria-roledescription`'s locale semantics correctly, the *ARIA Authoring Practices* should mention that authors  shall use locale-specific translations for "carousel" and "slide", and maybe even provide term suggestions for different languages.

Notice that the examples [*Auto-Rotating Image Carousel Example with Buttons for Slide Control*](https://www.w3.org/WAI/ARIA/apg/example-index/carousel/carousel-1-prev-next.html) and [*Auto-Rotating Image Carousel with Tabs for Slide Control Example*](https://w3c.github.io/aria-practices/examples/carousel/carousel-2-tablist.html) also currently use the English-centric `aria-roledescription` *carousel* and *slide* without mentioning that locale-specific terms should be used.

### PR DESCRIPTION
**Pull request by jongund**
Fixes #2355 by:

Adding a note to the carousel examples that the `aria-roledescription` value uses the language of the page, similar to labeling of controls.

[Auto-Rotating Image Carousel Example with Buttons for Slide Control](https://deploy-preview-111--wai-aria-practices2.netlify.app/aria/apg/example-index/carousel/carousel-1-prev-next.html)
[Auto-Rotating Image Carousel with Tabs for Slide Control Example](https://deploy-preview-111--wai-aria-practices2.netlify.app/aria/apg/example-index/carousel/carousel-2-tablist.html)

___
[WAI Preview Link](https://deploy-preview-111--wai-aria-practices2.netlify.app)

### CODE REVIEW

**Comment by jongund** [comment]
@mcking65 @jnurthen 
I was cleaning up my old pull requests and this was one from about a year ago for the carousel examples.  It adds an additional bullet to the Properties and States descriptions for the `aria-roledescription` property value to be in the language of the page.  It only adds 2 lines of content to each carousel example file.

File: content/patterns/carousel/examples/carousel-1-prev-next.html [hunk 1]
```diff
@@ -391,6 +391,7 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
                 <ul>
                   <li>Informs assistive technologies to identify the element as a &quot;carousel&quot; rather than a &quot;region.&quot;</li>
                   <li>Affects how the assistive technology renders the role but does not affect functionality, such as commands for navigating to landmark regions.</li>
+                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page.</li>
```
Comment (jnurthen) [inline content/patterns/carousel/examples/carousel-1-prev-next.html:394]:
```suggestion
                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page and should be localized by the author.</li>
```

File: content/patterns/carousel/examples/carousel-1-prev-next.html [hunk 2]
```diff
@@ -463,6 +464,7 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
                 <ul>
                   <li>Informs assistive technologies to identify the element as a &quot;slide&quot; rather than a &quot;group.&quot;</li>
                   <li>Affects how the assistive technology renders the role but does not remove any assistive technology functions related to <code>group</code> elements.</li>
+                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page.</li>
```
Comment (jnurthen) [inline content/patterns/carousel/examples/carousel-1-prev-next.html:467]:
```suggestion
                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page and should be localized by the author.</li>
```

File: content/patterns/carousel/examples/carousel-2-tablist.html [hunk 3]
```diff
@@ -499,6 +499,7 @@ <h3 id="rps_label_2">Tabs</h3>
                   <ul>
                     <li>Informs assistive technologies to identify the element as a "carousel" rather than as a "region".</li>
                     <li>Affects how assistive technologies render the role but does not affect functionality, such as commands for navigating to landmark regions.</li>
+                    <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page.</li>
```
Comment (jnurthen) [inline content/patterns/carousel/examples/carousel-2-tablist.html:502]:
```suggestion
                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page and should be localized by the author.</li>
```

File: content/patterns/carousel/examples/carousel-2-tablist.html [hunk 4]
```diff
@@ -676,6 +677,7 @@ <h3 id="rps_label_2">Tabs</h3>
                   <ul>
                     <li>Informs assistive technologies to identify the element as a "slide" rather than as a "tabpanel".</li>
                     <li>Affects how assistive technologies render the role but does not affect assistive technology functionality.</li>
+                    <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page.</li>
```
Comment (jnurthen) [inline content/patterns/carousel/examples/carousel-2-tablist.html:680]:
```suggestion
                  <li>NOTE: The <code>aria-roledescription</code> value is defined using the language of the page and should be localized by the author.</li>
```

**Review by jnurthen — APPROVED** [review]
I prefer adding the suggested text.

**Comment by jongund** [comment]
@jnurthen 
I added your suggest edit to include "... and should be localized by the author."

**Review by mcking65 — COMMENTED** [review]
Thank you @jongund, looks great, merging.

**Pull request merged**